packages/ember-metal/lib/platform/create.js

```diff
@@ -100,4 +100,4 @@ if (!(Object.create && !Object.create(null).hasOwnProperty)) {
   create = Object.create;
 }
 
-export default create;
+export default Object.create;
```

Comment:
Yes, we likely should.  We are still exporting `Ember.create = Object.create` for global export support, but that shouldn't require this file...